### ISSUE TITLE
VertexAISearchRetriever does not support unstructured data stores with chunking enabled.

### ISSUE THREAD

**Issue by akos-sch** [langchain-ai/langchain-google#229]
Recently, Google released [layout parser](https://cloud.google.com/generative-ai-app-builder/docs/parse-chunk-documents?_gl=1*94i227*_ga*MjQwNDk5MzkzLjE2NjIzNjYwMjU.*_ga_WH2QY8WWF5*MTcxNTkzMjc4Ni4yOTUuMC4xNzE1OTMyNzg2LjAuMC4w&_ga=2.113757444.-240499393.1662366025#layout-parsing) for data stores: _The layout parser is available only when using document chunking for RAG. When document chunking is turned on, Vertex AI Search breaks documents up into chunks at ingestion time and can return documents as chunks._

I created an unstructured data store with layout parsing enabled. I tried running the following snippet:
```
retriever = VertexAISearchRetriever(
    project_id=PROJECT_ID,
    location_id=DS_LOCATION_ID,
    data_store_id=DATA_STORE_ID,
    max_documents=10,
    engine_data_type=DATA_STORE_TYPE, # set to 0 for unstructured data store
)

retriever_tool = create_retriever_tool(
    retriever=retriever, 
    name=RETRIEVER_NAME, 
    description=RETRIEVER_DESCR
)

docs = retriever_tool.invoke({"query": question})
```
I receive the following error:
`google.api_core.exceptions.InvalidArgument: 400 `extractive_content_spec` must be not defined when the datastore is using 'chunking config' This might be due to engine_data_type not set correctly.`

As of 17/05/2024, this feature is brand new, and the corresponding [file ](https://github.com/langchain-ai/langchain-google/blob/main/libs/community/langchain_google_community/vertex_ai_search.py) was last modified 3 weeks ago, I assume the library has not tracked these changes yet.

I use version `"1.0.3"` of `langchain-google-community`.

Is there something I am missing here, or is my assumption correct?

**Comment by lkuligin** [comment]
yes, most probably support is lacking on the integration side. @jzaldi

**Comment by mdrxy** [comment]
Hello! Please let me know if this is still persisting in the latest version and I can re-open if so.
